modules/exploits/multi/handler.rb

```diff
@@ -30,7 +30,7 @@ def initialize(info = {})
           'BadChars'    => '',
           'DisableNops' => true,
         },
-      'Platform'       => %w{ android bsd java js linux osx nodejs php python ruby solaris unix win },
+      'Platform'       => %w{ android bsd java js linux osx nodejs php python ruby solaris unix win mainframe },
```

Comment:
is 'mainframe' the right word here, or should it be more specific to allow for other mainframe architectures? Do such things exist that we would care about?